index.js

```diff
@@ -4,24 +4,17 @@ const fs = require('fs');
 const env = require('jsdoc/env'); // eslint-disable-line import/no-unresolved
 const addInherited = require('jsdoc/augment').addInherited; // eslint-disable-line import/no-unresolved
 
-const config = env.conf.typescript;
-if (!config) {
-  throw new Error(
-    'Configuration "typescript" for jsdoc-plugin-typescript missing.',
-  );
-}
-if (!('moduleRoot' in config)) {
-  throw new Error(
-    'Configuration "typescript.moduleRoot" for jsdoc-plugin-typescript missing.',
-  );
-}
-const moduleRoot = config.moduleRoot;
-const moduleRootAbsolute = path.join(process.cwd(), moduleRoot);
-if (!fs.existsSync(moduleRootAbsolute)) {
+const config = env.conf;
+const moduleRoot = config.typescript ? config.typescript.moduleRoot : undefined;
+const moduleRootAbsolute = moduleRoot
+  ? path.join(process.cwd(), moduleRoot)
+  : undefined;
+
+if (moduleRootAbsolute && !fs.existsSync(moduleRootAbsolute)) {
   throw new Error(
     'Directory "' +
       moduleRootAbsolute +
-      '" does not exist. Check the "typescript.moduleRoot" config option for jsdoc-plugin-typescript',
+      '" does not exist. Check the "typescript.moduleRoot" config option for jsdoc-plugin-typescript'
```

Comment:
See the [contributing doc](https://github.com/openlayers/jsdoc-plugin-typescript/blob/main/contributing.md) for details on getting set up with the dependencies needed to conform with the linter rules in this project.  I added some additional details in https://github.com/openlayers/jsdoc-plugin-typescript/pull/58#issuecomment-2315931865 as well.

Reply:
I think `eslint-config-openlayers` needs `trailingCommas: 'all'` set to prevent this.